pkg/lib/cockpit.d.ts

```diff
@@ -335,7 +335,7 @@ declare module 'cockpit' {
         home: string;
         shell: string;
     };
-    export function user(): Promise<UserInfo>;
+    export function user(): Promise<Readonly<UserInfo>>;
```

Comment:
Should we also mark it as deprecated or is that why you say soft deprecate instead?
```suggestion
    /** @deprecated */ export function user(): Promise<Readonly<UserInfo>>;
```

Reply:
Added!